p2p/relay.go

```diff
@@ -30,6 +31,28 @@
 	"github.com/obolnetwork/charon/app/z"
 )
 
+// NewRelays returns the libp2p circuit relays from bootnodes if enabled.
+func NewRelays(conf Config, bootnodes []*enode.Node) ([]Peer, error) {
+	if !conf.BootnodeRelay {
+		return nil, nil
+	} else if conf.UDPBootManifest {
+		// Relays not supported via manifest bootnodes yet.
+		return nil, nil
```

Comment:
maybe add a unit test for these scenarios

Reply:
yeah